app/workers/find_and_delete_scheduled_accounts_worker.rb

```diff
@@ -4,7 +4,6 @@ class FindAndDeleteScheduledAccountsWorker
   include Sidekiq::Worker
 
   def perform
-    return unless ThreeScale.config.onpremises
```

Comment:
Right 👍 But for the tests to pass, you will have to delete this:
https://github.com/3scale/porta/blob/4b1d9bf7d223a27e74afc81fcc3a1612d1447f6c/test/workers/find_and_delete_accounts_scheduled_worker_test.rb#L12-L16

Reply:
right 🍳